files/build_templates/docker_image_ctl.j2

```diff
@@ -90,10 +90,12 @@ function preStartAction()
         if [[ ("$BOOT_TYPE" == "warm" || "$BOOT_TYPE" == "fastfast" || "$BOOT_TYPE" == "fast")  && -f $WARM_DIR/dump.rdb ]]; then
             # Load redis content from /host/warmboot/dump.rdb
             docker cp $WARM_DIR/dump.rdb database$DEV:/var/lib/redis/dump.rdb
+            docker cp $WARM_DIR/dump.rdb database$DEV:/var/lib/redis_bmp/dump.rdb
```

Comment:
>docker cp $WARM_DIR/dump.rdb database$DEV:/var/lib/redis_bmp/dump.rdb [](http://example.com/codeflow?start=12&length=69)

BGP feature has no special handling for Warm restart support. The code change here will hurt warm bootup process. Why changing?

Reply:
yes, we should not need this warm restart operation. removed.